src/type/type.js

```diff
@@ -180,7 +180,10 @@ export default function getType(graffitiModels, {name, description, fields}, roo
 
     const graphQLField = {name, description};
 
-    if (type === 'Array') {
+    if (type === 'Array' && subtype == 'Object') {
```

Comment:
double equal, also duplicate test of `type === 'Array'`, I would move it in one if